src/lib/server/file-data.ts

```diff
@@ -9,7 +9,7 @@ import { JsonFile, fileExists } from './fs-util';
 
 const DATA_DIR = './data/';
 const MODEL_DIR = join(DATA_DIR, 'models');
-const USERS_JSON = join(DATA_DIR, 'users.json');
+const USERS_JSON = join(DATA_DIR, 'users-discord.json');
```

Comment:
Why?

Reply:
Because I forgot to replace the original once I was finished 😅